seedpod_ground_risk/main.py

```diff
@@ -103,6 +106,7 @@ def mousePressEvent(self, event: PySide2.QtGui.QMouseEvent) -> None:
             menu.addAction("Edit Layer", self.edit_layer)
             if isinstance(self._layer, AnnotationLayer):
                 menu.addAction("Export .GeoJSON", self.export_path_json)
+                menu.addAction("Show path data", self.path_data)
```

Comment:
Show path data isn't a valid action for all `AnnotationLayer`s, there are other classes that inherit `AnnotationLayer` that wouldn't know how to/couldn't handle this action